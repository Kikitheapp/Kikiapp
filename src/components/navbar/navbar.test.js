import React from "react"
import renderer from 'react-test-renderer';

import Navbar from "./navbar"

describe("Navbar", () => {
  it("renders correctly", () => {
    const navbar = renderer
      .create(<Navbar />)
      .toJSON()
    expect(navbar).toMatchSnapshot()
  })
})