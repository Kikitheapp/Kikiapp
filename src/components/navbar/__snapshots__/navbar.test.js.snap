// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light"
  id="navbar"
>
  <div
    className="container-fluid"
  >
    <h1>
      <a
        className="navbar-brand"
        href="/"
      >
        <img
          alt="Kiki"
          src="test-file-stub"
        />
      </a>
    </h1>
    <button
      aria-controls="navbarSupportedContent"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-bs-target="#navbarSupportedContent"
      data-bs-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarSupportedContent"
    >
      <ul
        className="navbar-nav ms-auto"
      >
        <li
          className="nav-item dropdown"
        >
          <button
            aria-expanded="false"
            aria-haspopup="true"
            className="nav-link dropdown-toggle"
            data-bs-toggle="dropdown"
            href="#"
            id="navbarDropdown"
          >
            Get Started
          </button>
          <div
            aria-labelledby="navbarDropdown"
            className="dropdown-menu"
          >
            <a
              className="dropdown-item"
              href="/aboutus"
            >
              About Us
            </a>
            <a
              className="dropdown-item"
              href="/faq"
            >
              FAQ
            </a>
            <a
              className="dropdown-item"
              href="/prep"
            >
              PrEP Q&A
            </a>
            <a
              className="dropdown-item"
              href="/meetthesextoys"
            >
              Meet the sex toys
            </a>
            <a
              className="dropdown-item"
              href="/resources"
            >
              Resources
            </a>
          </div>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="https://kikiapp.medium.com/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Visit our blog
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="https://shop.kikitheapp.com/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Shop
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="/mailinglist"
          >
            Join our mailing list
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
